key.util/src/main/java/org/key_project/util/java/StringUtil.java

```diff
@@ -433,4 +433,24 @@ public static String trim(String text, String chars) {
     public static String replaceNewlines(String text, String with) {
         return NEWLINE_PATTERN.matcher(text).replaceAll(with);
     }
+
+    /**
+     * Counts occurrences of ch
+     *
+     * @param str the string
+     * @param ch what to look for
+     * @return the number of occurrences
+     */
+    public static int countMatches(String str, char ch) {
```

Comment:
I wonder if `countMatches` is the right name for the method.

When googling for a better name, I found that this code is taken from Apache commons.
That must be attributed. We probably have to mention this somewhere.

Reply:
/cc @wadoon. Do we have some license automation in the workflows?

Reply:
Yes that was taken from Apache Commons and should be named `countOccurences`. Do we really need to attribute this? It's 5 LOC that would have looked exactly the same if I just typed them by hand.

Reply:
There are plugins to check for license violatons, see https://github.com/jk1/Gradle-License-Report, but this does not help in these case, as this is not a Gradle dependency. 

In this case, a note in the Javadoc should be enough to give credits to Apache Commons. Apache License should be compatible with GPLv2.

Reply:
Removed the function since it was unused. I think we should just include apache commons, since it has a lot of useful functions.